Lib/fontTools/cu2qu/errors.py

```diff
@@ -19,7 +19,7 @@ class Error(Exception):
 class ApproxNotFoundError(Error):
     def __init__(self, curve):
         message = "no approximation found: %s" % curve
-        super(Error, self).__init__(message)
+        super(ApproxNotFoundError, self).__init__(message)
```

Comment:
or just this, since we're py3 only now
```suggestion
        super().__init__(message)
```